cfgov/unprocessed/js/organisms/FeaturedContentModule.js

```diff
@@ -22,46 +19,52 @@ let _videoPlayer;
  * @returns {FeaturedContentModule} An instance.
  */
 function FeaturedContentModule( element ) {
-  _dom = checkDom( element, BASE_CLASS );
+  const _dom = checkDom( element, BASE_CLASS );
 
-  this.init = init;
+  let _videoPlayer = null;
 
-  return this;
-}
+  /**
+   * @returns {FeaturedContentModule|undefined} An instance.
```

Comment:
Eventually we should update these init methods to just return the instance instead of undefined if the instance has already been initialized. I put in undefined so you'd have a way to know if something was already initialized, but thinking about it further I think it should always just return the instance. This should happen with all code structures like this across the codebase at once, so we should probably leave it alone here.